readme.md

```diff
@@ -1015,6 +1015,18 @@ If you liked Cloud Atlas this is a good read.
 
 _Novels which focus on the near-future unintended consequences of biotechnology revolutions._
 
+#### [Pandemic](https://www.goodreads.com/book/show/34811896-pandemic) (2017) *by [A.G. Riddle](https://www.goodreads.com/author/show/3087199.A_G_Riddle)
```

Comment:
Bold block needs to be closed.

Reply:
Sorry, been a bit busy, I'll get it corrected tonight when I get home.

Reply:
I think I figured it out, should be fixed now.